File: answer.py
def outed(meet, boss):
    return 'Get Out Now!' if (sum(meet.values()) + meet[boss] ) / len(meet) <= 5 else 'Nice Work Champ!'
  
def outed(meet, boss):
    total_person = len(meet.keys())
    total_score = 0
    for x in meet.values(): total_score += x
    return 'Nice Work Champ!' if (total_score + meet[boss])/total_person > 5 else 'Get Out Now!'

File: answer.js
function outed(meet, boss) {
  let names = Object.keys(meet)
  let score = names.reduce((s,v) => s + meet[v], 0) + meet[boss]
  return score / names.length > 5 ? 'Nice Work Champ!' : 'Get Out Now!'
}

const outed =(meet,boss)=>{
    let totalNumbersOfPeople=Object.keys(meet).length
    let totalScore = 0
    for(let attendace in meet){
        attendace==boss ? totalScore+=meet[attendace]*2 :totalScore+=meet[attendace]
    }
    return Math.round(totalScore/totalNumbersOfPeople)>5 ? "Nice Work Champ" : "Get out now"
}
